### PR TITLE
ark: update placeholders in translation

### PR DESCRIPTION
**Pull request by kbdharun**
Continuation of https://github.com/tldr-pages/tldr/pull/9541

### CODE REVIEW

**Review by einverne — APPROVED** [review]
zh pages look good to me.

**Review by pixelcmtd — APPROVED** [review]
de LGTM

File: pages.zh/linux/ark.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
 
 - 改变解压缩目录：
 
-`ark --batch --destination {{解压缩目录路径}} {{存档名}}`
+`ark --batch --destination {{路径/到/目录}} {{路径/到/存档名}}`
 
 - 创建一个原本不存在的存档并向它添加文件：
 
-`ark --add-to {{存档名}} {{文件1}} {{文件2}}`
+`ark --add-to {{路径/到/存档名}} {{路径/到/文件1个 路径/到/文件2个 ...}}`
```
Comment (blueskyson) [inline pages.zh/linux/ark.md:16]:
```suggestion
`ark --add-to {{路径/到/存档名}} {{路径/到/文件1 路径/到/文件2 ...}}`
```

Comment (kbdharun) [reply pages.zh/linux/ark.md]: I found similar occurrences of this on other pages too, I will update it in a new PR.

Comment (kbdharun) [reply pages.zh/linux/ark.md]: I found similar occurrences of this on other pages too, I will update it in a new PR.

**Pull request merged**